### ISSUE TITLE
Documentation improvements - structure / organization

### ISSUE THREAD

**Issue by lordpretzel** [VizierDB/web-ui#295]
- add submenus for the wiki to organize topics hierarchically, e.g., [https://github.com/IITDBGroup/gprom/wiki](https://github.com/IITDBGroup/gprom/wiki). Some initial suggestion (to be discussed):
  - Overview & Concepts
  - Installation & Setup
  - Usage
  - Development
  - Cell Documentation
- pages that we may want to add:
  - API documentation
  - list of tutorials / how-tos (add over time for now the getting started one falls into this cateogry)
    - how to get data into Vizier
  - Vizier architecture (for developers and technical users that want to understand at a high-level of how Vizier works)
- home
  - possibly add a short description of what Vizier is

**Comment by okennedy** [comment]
Quick clarification on the first bullet: is the suggestion here to organize the sidebar along the same lines as the main page ( https://github.com/VizierDB/web-ui/wiki )?

How to get data into Vizier is definitely a great idea.  I'll put that together. 

API documentation is a bit of a work in progress.  There's work on Swagger API generator in progress.

Vizier architecture... perhaps some pointers to papers?

Agreed on the overview on the home page.

**Comment by lordpretzel** [comment]
Right, to just have some hierarchical structure in the side menu. Not sure what the best grouping is, but we can start with the one from the main page.

About the architecture, just a very brief description of what are the components of vizier (maybe one of the graphics from our papers) and what the ecosystem is (written in Scala, ...)

**Comment by okennedy** [comment]
I think all of these should now be applied.
